### PR TITLE
Update montaging with missing files

### PR DESCRIPTION
**Pull request by jni**
We had this functionality but it hardcoded some screen settings. This PR opens them up so we can use these functions on new screens.

### CODE REVIEW

File: microscopium/preprocess.py [hunk 1]
```diff
@@ -910,7 +910,7 @@ def create_missing_mask(missing, order, rows=512, cols=512):
     return mask
 
 
-def montage_with_missing(fns, order=None):
+def montage_with_missing(fns, *, order, re_string, re_group):
     """Montage a list of images, replacing missing fields with dummy values.
```
Comment (GenevieveBuckley) [inline microscopium/preprocess.py]: What are the dummy values, NA?

Comment (jni) [reply microscopium/preprocess.py]: No, dummy = 0 at this point. Anything else will wreak havoc with the feature computation. The function also returns a mask but currently we can't use it.

**Comment by jni** [comment]
@GenevieveBuckley merging because I need this to get the production run going on MASSIVE. Let me know if you want me to change things in a future PR.

File: microscopium/preprocess.py (hunk 1 above)
Comment (GenevieveBuckley) [reply microscopium/preprocess.py]: > The function also returns a mask but currently we can't use it.

Uh, why?

**Comment by GenevieveBuckley** [comment]
Nice work :)

File: microscopium/preprocess.py (hunk 1 above)
Comment (jni) [reply microscopium/preprocess.py]: I meant, we don’t have any functions (specifically feature computation functions) that can take in masks as input.

**Pull request merged**